docs/source/modules/idealised_moist_phys.rst

```diff
@@ -0,0 +1,295 @@
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``land_field_name``         |Field name in the input land-mask netcdf.                             |``'land_mask'``    |
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``bucket``                  |If true, use bucket hydrology.                                        |``False``          |
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``init_bucket_depth``       |Value at which to initialise bucket water depth over ocean (large).   |``1000.``          |
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``init_bucket_depth_land``  |Value at which to initialise bucket water depth over land.            |``20.``            |
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``max_bucket_depth_land``   |Maximum depth of water in bucket over land following intialisation.   |``0.15``           |
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``robert_bucket``           |Robert coefficient for RAW filter on bucket leapfrog timestepping.    |``0.04``           |
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``raw_bucket``              |RAW coefficient for RAW filter on bucket leapfrog timestepping.       |``0.53``           |
++----------------------------+----------------------------------------------------------------------+-------------------+
+
+``land_option`` may be set to:
+
++---------------+------------------------------------------------------------------------------------------+
+|Value          | Effect                                                                                   |
++===============+==========================================================================================+
+|``'input'``    |Read land mask from input file.                                                           |
++---------------+------------------------------------------------------------------------------------------+
+|``'zsurf'``    |Define land where surface geopotential height at model initialisation exceeds a threshold.|
++---------------+------------------------------------------------------------------------------------------+
+|``'none'``     | Do not apply a land mask                                                                 |
++---------------+------------------------------------------------------------------------------------------+
+
+
+
+									  
+Diagnostics
```

Comment:
Suggestion: math-mode the entire units given in this table, instead of only the ^{-1} etc